.github/workflows/run-e2e-tests.yaml

```
@@ -0,0 +1,102 @@
+name: run e2e tests
+'on':
+  schedule:
+    # run twice a day at 10:00 and 22:00 UTC every day of the week
+    - cron: "0 10/12 * * *"
+  workflow_dispatch:
+env:
+  BEE_NAME: 'wds-${{ github.run_id }}-${{ github.run_attempt}}-dev'
+  TOKEN: '${{ secrets.BROADBOT_TOKEN }}'
+  RUN_NAME_SUFFIX: >-
+    ${{ github.event.repository.name }}-${{ github.run_id }}-${{
+    github.run_attempt }}
```

Could this all go on one line?

```suggestion
  RUN_NAME_SUFFIX: '${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}'
```

---

Yeah not sure what happened here